felix/bpf/proxy/syncer.go

```diff
@@ -207,6 +219,12 @@ func NewSyncer(nodePortIPs []net.IP,
 		prevSvcMap:  make(map[svcKey]svcInfo),
 		prevEpsMap:  make(k8sp.EndpointsMap),
 		stop:        make(chan struct{}),
+
+		newFEK:       newFEK,
```

Comment:
I wouldn't mind these being more verbose `s.newFrontEndKey(...)` feels pretty natural and will be easier for newcomers to grok.

Reply:
residual of the FEK etc. based generics...